src/lib/y2partitioner/widgets/rescan_devices_button.rb

```diff
@@ -68,12 +69,17 @@ def rescan_message
         end
       end
 
-      # Reprobes and updates devicegraphs for the partitioner
+      # Reprobes and updates devicegraphs for the partitioner.
       #
-      # @note A message is shown during the reprobing action
+      # @note A message is shown during the reprobing action.
+      #
+      # @raise [Y2Partitioner::ForcedAbortError] When the probed devicegraph contains errors
+      #   and the user decices to not sanitize
```

Comment:
typo: decices.

Moreover, the exception is also raised if there is an error during probing itself and the user decides to abort (so not only sanitization is affected).